tox.ini

```diff
@@ -9,6 +9,9 @@ commands =
 deps = -rdev-requirements.txt
 install_command = pip install --pre {opts} {packages}
 
+[testenv:functional]
+commands = xml2kinto -s http://localhost:8888/v1 -x blocklist.xml -S schemas.json
```

Comment:
that could also be a way to do it. It means that it'll install tox and create a venv for it, but why not. It makes sense to have both use the same "code" though, to make sure we don't introduce incoherences in the future, or experience a different behavior on travis and dev.